src/patternfly/components/Title/styles.scss

```diff
@@ -0,0 +1,119 @@
+@import "../../patternfly-utilities";
+
+// xxxxl
+$pf-c-title--xxxxl--MarginTop: $pf-global--spacer--lg !default;
```

Comment:
we are reproducing the name of the class: this `pf-c-title--xxxxl--MarginTop` should be `pf-c-title--is-xxxxl--MarginTop`, it's missing the `is-` on all of them.

Do we need a comment on the guidelines to specify this?

Reply:
That might be a good idea